src/index.jsx

```diff
@@ -42,8 +40,7 @@ function App() {
           <Switch>
             <Route path="/" exact component={LandingPage} />
             <Route path="/table" component={TablePage} />
-            <Route path="/example-list" component={ExampleListPage} />
-            <Route path="/datavis" component={ExampleDataViz} />
+            <Route path="/heatmap" component={NotFoundPage} />
```

Comment:
I like how this enables future components to be implemented in the menu before being finished -- it makes the site feel more complete without it yet being completed.